daemon/create.go

```diff
@@ -16,6 +17,13 @@ func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hos
 		return "", warnings, err
 	}
 
+	// The check for a valid workdir path is made on the server rather than in the
+	// client. This is because we don't know the type of path (Linux or Windows)
+	// to validate on the client.
+	if config.WorkingDir != "" && !filepath.IsAbs(config.WorkingDir) {
```

Comment:
@jhowardmsft I didn't know runconfig.Parse was client side. For example when I create a container with an invalid name, it says daemon has responded with error:

![image](https://cloud.githubusercontent.com/assets/159209/7376922/fd403c1a-ed97-11e4-8e96-e76c6833aba4.png)

wherever this name check is going on, could that be a good place to add this check also?